### PR TITLE
fix: rename pfd -> pfb for celestia-node

### PR DESCRIPTION
**Pull request by distractedm1nd**
## Overview
Accidentally based on #437 , so will rebase on main once it is merged.
This updates the naming of PayForData to PayForBlob, which is a change that will be included in the next breaking celestia-node release (v0.70)

Resolves https://github.com/celestiaorg/docs/issues/459

Related: https://github.com/celestiaorg/celestia-node/pull/1582

The PR

### CODE REVIEW

**Comment by jcstein** [comment]
note: deploy PR is failing, because it came from a fork/branch outside the repo

**Comment by jcstein** [comment]
also, i tested out the transactions in order to provide up-to-date examples in the case of PFBs

File: docs/developers/node-tutorial.mdx [hunk 1]
```diff
@@ -749,8 +749,8 @@ celestia light start --core.ip <ip-address> --keyring.accname <key_name> --gatew
 ````
 
 Once you start the Light Node, a wallet key will be generated for you.
-You will need to fund that address with Arabica Devnet tokens to pay
-for PayForData transactions.
+You will need to fund that address with Mocha Testnet or Arabica Devnet
+tokens to pay for `PayForData` transactions.
```
Comment (YazzyYaz) [inline docs/developers/node-tutorial.mdx:753]: This doesn't seem true for both. One requires PFD, the other PFB

File: docs/nodes/bridge-node.mdx [hunk 2]
```diff
@@ -173,29 +192,28 @@ Here is an example of starting the bridge node:
...
-You have two networks to get testnet tokens from:
+You can get testnet tokens from:
 
-* [Arabica](./arabica-devnet.md)
-* [Mocha](./mocha-testnet.md)
+* [Arabica](../arabica-devnet)
 
 > NOTE: If you are running a bridge node for your validator
-  it is highly recommended to request Mocha testnet tokens
+  it is highly recommended to request Arabica devnet tokens
```
Comment (YazzyYaz) [inline docs/nodes/bridge-node.mdx:211]: This is false. Arabica isn't used to test out validator operations

File: docs/nodes/bridge-node.mdx [hunk 3]
```diff
@@ -173,29 +192,28 @@ Here is an example of starting the bridge node:
...
 
 ```sh
 ./cel-key list --node.type bridge --keyring-backend test --p2p.network <network>
 ```
 
-You have two networks to get testnet tokens from:
+You can get testnet tokens from:
 
-* [Arabica](./arabica-devnet.md)
-* [Mocha](./mocha-testnet.md)
```
Comment (YazzyYaz) [inline docs/nodes/bridge-node.mdx:194]: Why are removing Mocha from here?

File: docs/nodes/full-storage-node.mdx [hunk 4]
```diff
@@ -88,50 +92,84 @@ A note on ports:
   after the IP address or use the  `--core.grpc.port` flag to specify another
   port if you prefer.
 
-````mdx-code-block
-<Tabs groupId="network">
```
Comment (YazzyYaz) [inline docs/nodes/full-storage-node.mdx:92]: Why are tabs being removed from here?

File: docs/nodes/full-storage-node.mdx [hunk 5]
```diff
@@ -88,50 +92,84 @@ A note on ports:
...
-<TabItem value="mocha" label="Mocha">
-
 ```sh 
-celestia full start --core.ip <ip-address>
+celestia full start --core.ip <ip-address>:<port>
 ```
 
-</TabItem>
-<TabItem value="arabica" label="Arabica 🏗️">
```
Comment (YazzyYaz) [inline docs/nodes/full-storage-node.mdx:100]: Why are tabs being removed from here?

File: docs/nodes/full-storage-node.mdx [hunk 6]
```diff
@@ -88,50 +92,84 @@ A note on ports:
...
+If you would like to find example RPC endpoints, check out the list of
+resources [here](../mocha-testnet).
```
Comment (YazzyYaz) [inline docs/nodes/full-storage-node.mdx:100]: [here](./mocha-testnet)

File: docs/nodes/bridge-node.mdx (hunk 3 above)
Comment (jcstein) [reply docs/nodes/bridge-node.mdx:194]: because there is now a tab for each network

File: docs/nodes/full-storage-node.mdx (hunk 4 above)
Comment (jcstein) [reply docs/nodes/full-storage-node.mdx:92]: fairly certain i was just rearranging where the tabbed section starts

File: docs/nodes/full-storage-node.mdx (hunk 5 above)
Comment (jcstein) [reply docs/nodes/full-storage-node.mdx:100]: same as above. this is an old PR and I had to rearrange the order of the tabs, the same way that they are on the live site. where mocha is first, and arabica second

File: docs/nodes/full-storage-node.mdx (hunk 6 above)
Comment (jcstein) [reply docs/nodes/full-storage-node.mdx:100]: yo soy confused! what do you mean?

File: docs/developers/node-tutorial.mdx (hunk 1 above)
Comment (jcstein) [reply docs/developers/node-tutorial.mdx]: 2d4a476bf7f96b1a30a321e8ed011dfc0eb2f466

File: docs/nodes/full-storage-node.mdx (hunk 4 above)
Comment (jcstein) [reply docs/nodes/full-storage-node.mdx:92]: yea, they're not removed they have just been moved and cover more of the page. resolving

File: docs/nodes/full-storage-node.mdx (hunk 6 above)
Comment (YazzyYaz) [reply docs/nodes/full-storage-node.mdx:100]: updated comment

Comment (YazzyYaz) [reply docs/nodes/full-storage-node.mdx:100]: Actually not sure if this makes a difference given you're using a certain style for links.

**Review by YazzyYaz — APPROVED** [review]

File: docs/nodes/full-storage-node.mdx (hunk 6 above)
Comment (jcstein) [reply docs/nodes/full-storage-node.mdx:100]: 😅🫡

**Pull request merged**